src/util/DeviceFingerprint.js

```diff
@@ -23,6 +26,7 @@ define(['vendor/lib/q', 'okta/jquery'], function (Q, $) {
       }
 
       var deferred = Q.defer();
+      var self = this;
```

Comment:
why do you need to do this? Can't you just call `this.METHOD` or is this a weird js thing?

Reply:
cool thanks for clarifying 👍